src/config_handling.py

```diff
@@ -43,7 +52,15 @@ def init_config(app):
             sys.exit(1)
         app.config[config] = value
 
-    for config in all_configs:
-        if app.config.get(config) is None:
-            app.logger.critical(f"Did not find an environment variable for '{config}'")
-            sys.exit(1)
+
+def get_config_from(app, config):
+    path = os.environ.get(config)
+    if not path:
+        app.logger.critical(f"Did not find environment variable for '{config}'")
+        sys.exit(1)
+    try:
+        with open(path) as file_:
+            return file_.read()
+    except Exception:  # noqa
+        app.logger.critical(f"Reading path '{path}' from config '{config}' went wrong.")
+        sys.exit(1)
```

Comment:
One hint: 
If we want a similar logic as in our datastore, then you can write something like this:

```suggestion
# Somewhere at the top of this file
DEV_SECRET = "openslides"

# [...]
def get_config_from(config_name: str) -> str:
  if is_dev_mode():
    value = DEV_SECRET
  elif path := os.environ.get(config_name + "_FILE", None) is not None:
    with open(path) as file_:
      value = file_.read()
  else:
    value = os.environ.get(config_name, None)
  return value
```

If `is_dev_mode` is not already implemented, you have also to implement this function.

Reply:
Implement is_dev_mode, add this var to the test docker compose, update the var get mechanism.